rhino/src/main/java/org/mozilla/javascript/IRFactory.java

```diff
@@ -815,7 +855,7 @@ private Node transformNumber(NumberLiteral node) {
         return node;
     }
 
-    private Node transformObjectLiteral(ObjectLiteral node) {
+    public Node transformObjectLiteral(ObjectLiteral node) {
```

Comment:
Not sure why this had to become public, I don't see any other uses within the diff.

Reply:
Will fix!